discovery-streams/values.yaml

```diff
@@ -29,3 +29,6 @@ smokeTestImage:
   repository: smartcitiesdata/discovery_streams_smoke_test
   tag: development
   pullPolicy: Always
+
+redis:
+  host: discovery-streams-redis-master
```

Comment:
does it make sense to have this "default" value be what we expect the uber chart to deploy? something like "redis.external-services"? This feels like an arbitrary value.

Reply:
I've wrestled with this too.  I think at this point the standard is that arbitrary value that is overridden, but I feel like it should be a default that tends towards working with our system (like `redis.external-services`). Seems like this is something we can cleanup/adjust as one change in charts at some point.

Reply:
In my mind `redis.external-services` is a Columbus specific implementation that should not be exposed here because it is not included in the uber chart.

Reply:
I think it should be. I think all the various internal services that use redis should be able to statically look for `redis.external-services` (or some other name abstraction) and then the uber chart always creates that record. The default value would point to an internal redis container service but if you flip a toggle that says “this redis is a managed service” then you have to provide the endpoint for that service and it gets written to as the backend for the external-service record. 

That way, you only have to update a single variable to point your cluster at its dependencies like redis, kafka, ldap, etc. currently we have to pass those endpoints to every service and ensure they get updated each time we deploy.